2025 updates (#73)
data-ethics-and-society/data-ethics-and-society-reading-group: Guides/organising.md

## Deciding on material to read

> The [reading list](/READING-LIST.md) is a curated list of what we, and others, have added over the years. It also shows what we have currently read, [more detail about previously discussed books is in our Sessions Overview](/SESSIONS.md).

Reviewer: ```suggestion
The [reading list](/READING-LIST.md) is a curated list of suggestions we have received, and indicates what we have covered in sessions. More detail about previously discussed books is in our [Sessions Overview](/SESSIONS.md).
```

> From 2025, the organisers will each pick a book to read and decide which one should be part of a future session. In general, we decide based on the following:

Reviewer: Can we add NOTE: here? (just to show that this is a point-in-time amendment)

> - Is the material engaging and interesting?

Reviewer: ```suggestion

- Is the material engaging and interesting?
```

> - Have we covered the topic before?

Reviewer: ```suggestion
- Is the content novel, or substantively different from material covered in previous sessions?
```

> - Is the book affordable and available?

Reviewer: ```suggestion
- Is the book affordable and widely available for open access, purchase or borrowing in the UK?
```

- Is the information and material up-to-date?

## Administrative tasks

More information on the tools we use can be found in our ['Tools'](./tools.md) page.

### Before the meeting

> - [ ] Bring suggestions to organiser meetings, divide material up between organisers, and decide which material should be discussed at the next event

Reviewer: I think having all these together implies it is a shorter task than it is, maybe split?
```suggestion
- [ ] Bring suggestions to organiser meetings and draw up a shortlist of material
- [ ] Divide material up between organisers and achieve consensus on which material should be discussed at the next event
```

- [ ] Decide date of next meeting
- [ ] Create a [meeting information file](./../Sessions/session-template.md) in the `Sessions/` folder
- [ ] Update the [meeting overview file](./../SESSIONS.md) with a link to the new meeting information page

> - [ ] Create a sign-up form to manage attendees (using Ticket Tailor)

Reviewer: can we not list the tools here? can we link to our tools page instead? also below re newsletter

Reviewer: and event- it just means fewer changes to make in future

We welcome any administrative and organisational help you can offer! Please get in touch via [email](mailto:xgov-data-ethics@proton.me). We'd especially like to hear from people who are new to the Civil Service and from groups usually under-represented in data science.

### I'd like to lead a session on a particular topic or book

Brilliant! Please reach out to us via [email](mailto:xgov-data-ethics@proton.me) and we can set this up with you.

## Communicating

> We primarily use our Substack [newsletter](https://xgovdataethics.substack.com/) to communicate with interested Data Science practitioners across government. Sign up to hear about upcoming events.

Reviewer: Higher level to avoid tooling content? Can link to newsletter section of tooling page?

Reviewer: think we should remove data science here as we are now more wide reaching
